frontend/package.json

```diff
@@ -60,6 +60,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "1.0.219",
+    "@patternfly/react-charts": "^2.2.3",
```

Comment:
this will cause a lot of complications while merging upstream
I'll try to fix it.